### ISSUE TITLE
are you seriously not aware about termux?

### ISSUE THREAD

**Issue by aicynide** [anacrolix/cove#8]
No need of Cove apk, it can be run in termux, just add Cove in termux packages repo https://github.com/termux/termux-packages
Termux has ffmpeg so there will be transcode support for android also

**Comment by anacrolix** [comment]
No! Thanks. I'll see if I can get a build for termux. Currently the apk runs the webserver but you have to connect to it manually from your browser. Is there anything I can do from termux to open the browser automatically?

**Comment by aicynide** [comment]
Yes why not. You can use `termux-open-url` see the example here https://github.com/reisxd/revanced-builder/blob/e1acfa7ff0359e8f078c7a5513299fc745286ceb/index.js#L57
Btw submit your cove package here https://github.com/termux/termux-packages

**Comment by aicynide** [comment]
Besides you can launch a chrome tab directly from termux with a shell command using `am` package. See the example here https://github.com/pystardust/ani-cli/blob/master/ani-cli#L248

**Comment by anacrolix** [comment]
Thanks

**Comment by master255** [comment]
Why aren't you using programs that are already available?

**Comment by aicynide** [comment]
Schizo

**Comment by anacrolix** [comment]
I took a look at termux. I didn't see a trivial way to contribute binaries, and wasn't able to test equivalent packages like transmission due to issues with linking.

**Comment by anacrolix** [comment]
I don't think I will create a termux package in the short term. You are welcome to package the binaries mentioned 
 in #11 and make them available on termux.

**Comment by anacrolix** [comment]
My understanding is that you cannot invoke processes on Android, which means transcoding and metadata detection is tricky. I need more specific feedback to confirm this and make a workaround.

I did not find working with termux to be a pleasant experience, but am happy to help out if someone wants to package cove for termux.

**Comment by anacrolix** [comment]
I tried termux later, I really like it. I'm still not sure how to contribute a binary for it, I might revisit this in the future.

I'm not sure cove in its current architecture is suited for running on typical phone hardware, it's probably far too heavy on disk I/O.
